README.md

```diff
@@ -374,7 +374,7 @@ RFC: [Unicode Codepoint Escape Syntax](https://wiki.php.net/rfc/unicode_escape)
 ### Closure call() Method
 
 The new `call()` method for closures is used as a shorthand way of invoking a
-closure whilst binding an object scope to it. This creates more perfomant and
+closure whilst binding an object scope to it. This creates more performance and
```

Comment:
_Creates more performance_ doesn't make sense. I suggest changing it to the follow:

```
This increases performance and compacts the code ...
```